recipes/perl-vcftools-vcf/0.840/run_test.pl

```diff
@@ -0,0 +1 @@
+use Vcf;
```

Comment:
Oh, is this broken in general? If so can you open an issue at conda-build with a minimal example?